### ISSUE TITLE
homepage social button 

### ISSUE THREAD

**Issue by omelki** [pro-freelancer/TOB#11]
The social media buttons should appear like this in this link when you hover on the recipe photos: http://foodgawker.com/

You can find the materials in the shared dropbox folder : https://www.dropbox.com/sh/43bjfdzkohvn3bn/AABtr7Vzgp_UNo2BwCNPb_TAa?dl=0

here is an example done in photoshop :

(https://cloud.githubusercontent.com/assets/7605683/6656992/11b785d8-cb45-11e4-9bf9-acb3de38e898.jpg)

-Note that on mobile you don't hover on the image anymore, the bar with the buttons should be fixed

**Comment by omelki** [comment]
On mobile the strip should be fixed coz you cannot hover with your finger

**Comment by pro-freelancer** [comment]
Start working on it @ 12:10 AM April 07 2015 and will try to deliver the HTML5 demo with a link tonight.

**Comment by pro-freelancer** [comment]
Finished at 2:15 AM.
http://pro-freelancer.github.io/TOB/homepage/

check the first , this is the transaction effects include your svg icons. Feedback is appreciated before implementing on real site.

after feedback , should fix some responsive CSS media query to fit all  screens, then implementation to the website.

**Comment by omelki** [comment]
It's a good start.

Small tweaks like make the animation faster use this :


   1. transition: 0.5s;
   2. -webkit-transform: translate3d(0, 0, 0);


like in this website: " http://foodgawker.com/ " the animation should be
smoother.

One more thing, the brown rectangle should appear from behind the white
rectangle and not on top of it.

thanks,

**Comment by pro-freelancer** [comment]
this was fixed for good
check the url.
http://pro-freelancer.github.io/TOB/homepage/
in case is approved will implement tomorrow on the website.

**Comment by omelki** [comment]
Yes It's good (: You can implement

**Comment by pro-freelancer** [comment]
I implement the icons but it seems that we didn't take into concideration that some images are long and all are not the same square sizes, wish caused the above CSS written is not good at all for the implementation. I revert the changes, I will redo the HTML css thing taking into consideration the other images cases such long images portrait to adopt a CSS solution for them.

**Comment by pro-freelancer** [comment]
I fixed the social button to be from bottom instead top, wish will work on any kind of images.
http://pro-freelancer.github.io/TOB/homepage/

will implement it tomorrow on wordpress using the staging system.

**Comment by omelki** [comment]
i can only see it working on the first post

**Comment by pro-freelancer** [comment]
yes it is the same demo that I fix to work from bottom instead from top. I should implement yesterday but still don't have internet at home, so Today will implement on the stanging and you will see it on all including exception displays.

**Comment by omelki** [comment]
man leh ma betzabbit el internet 3ndak

**Comment by pro-freelancer** [comment]
today will have internet for sure.
